### PR TITLE
Fix period marker issue.

### PR DESCRIPTION
**Pull request by kirsty-unosquare**
Fixes a bug with period marker. Needed an extra check to make sure dates between start and end date don't use the single event maker.

### CODE REVIEW

**Review by William-Unosquare — APPROVED** [review]
👍

File: NativeApp/src/components/Home/CustomDay/index.js [hunk 1]
```diff
@@ -28,19 +36,29 @@ const CustomDay = (props) => {
     );
   };
 
+  const renderFiller = (date) => {
+    let filler;
+    if (!sameDate) {
+      if (date || (!startingDate && !endingDate)) {
```
Comment (d1wilko) [inline NativeApp/src/components/Home/CustomDay/index.js:42]: is there a need for 2 separate if statements? - not a huge deal, just might be cleaner as one

**Review by d1wilko — APPROVED** [review]
just one question, but approving anyway

File: NativeApp/src/components/Home/CustomDay/index.js (hunk 1 above)
Comment (kirsty-unosquare) [reply NativeApp/src/components/Home/CustomDay/index.js:42]: It was really for my own benefit, easier to read while I was working on it. But can change to a one liner. :)

Comment (William-Unosquare) [reply NativeApp/src/components/Home/CustomDay/index.js:42]: @kirsty-unosquare I think it's easier to read this way aswell.

**Review by lee-cheung — APPROVED** [review]

**Pull request merged**